src/method/method_invoke_java.cpp

```diff
@@ -187,6 +187,15 @@ namespace cubmethod
     value_unpacker.value = &returnval;
     value_unpacker.unpack (unpacker);
 
+    if (db_value_type (&returnval) == DB_TYPE_RESULTSET)
+      {
+	std::uint64_t query_id = db_get_resultset (&returnval);
+	query_cursor *cursor = m_cursor_map [query_id];
+	cursor->close ();
```

Comment:
Is there any possibility that cursor is null?

Reply:
I've fixed